yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -126,11 +126,12 @@
               <t>A "status" "deprecated" statement MAY be added, or changed from "current" to "deprecated", but adding or
               changing "status" to "obsolete" is a non-backwards-compatible change.</t>
               <t>YANG schema nodes with a "status" "obsolete" substatement MAY be removed from published modules, and the removal  is classified as a backwards-compatible change.  In some circumstances it may be helpful to retain the obsolete definitions since their identifiers may still be referenced by other modules and to ensure that their identifiers are not reused with a different meaning.</t>
-              <t>A statement that is defined using the YANG "extension" statement MAY be added, removed, or changed, if it does not change the semantics of the module.  Extension statement definitions SHOULD specify whether adding, removing, or changing statements defined by that extension are backwards-compatible or non-backwards-compatible.</t>
+              <t>A statement that is defined using the YANG "extension" statement may be added, removed, or changed. While extensions may be ignored by some clients, other clients may support a given extension and thus be affected by changes to its semantics. Therefore, extension statement definitions SHOULD specify whether adding, removing, or changing statements defined by that extension are considered backwards-compatible or non-backwards-compatible.</t>
```

Comment:
replace "by changes to its semantics" with "if a module adds, removes or changes how it uses the extension statement."